README.md

```diff
@@ -370,6 +370,19 @@ docker).
 A previous version of this action did things with the wiki. This is not the case
 anymore.
 
+## .coverage file generated on a non-unix file system
+
+If your project needs to be built and tested on a non-unix os adding 
```

Comment:
`````suggestion
If your project's coverage was built on Windows, you may get an error like:
```
CoverageWarning: Couldn't parse 'yourproject\__init__.py': No source for code: 'yourproject\__init__.py'. (couldnt-parse)
```
This is likely due to coverage being confused with the coverage being computed with `\` but read with `/`. You can most probably fix it with the following in your [coverage configuration](https://coverage.readthedocs.io/en/latest/config.html):
`````